### PR TITLE
chore: update React and package versions

### PR DESCRIPTION
**Pull request by adamgrharvey**
https://github.com/fullboar/vc-showcase/issues/2

Updates React and all supporting packages to the latest versions.

Includes refactoring, where necessary, to fix errors related to version upgrades.

### CODE REVIEW

**Comment by adamgrharvey** [comment]
@weslord

File: .eslintrc.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ module.exports = {
     },
   },
   rules: {
+    '@typescript-eslint/no-explicit-any': 'off',
```
Comment (weslord) [inline .eslintrc.js:29]: Making a note to revisit this properly at some point

**Review by weslord — APPROVED** [review]

**Comment by adamgrharvey** [comment]
@jleach Hey Jason, this is ready for your review

**Comment by adamgrharvey** [comment]
Resolved merge conflicts.

**Pull request merged**